src/snek5000/output/base.py

```diff
@@ -294,6 +301,12 @@ def update_snakemake_config(
 
             append_debug_flags(config, warnings)
 
+            if env_sensitive is None:
+                env_sensitive = os.environ.get("SNEK_UPDATE_CONFIG_ENV_SENSITIVE", False)
+                if isinstance(env_sensitive, str):
+                    # correct for "0", "false", "False"
+                    env_sensitive = bool(yaml.safe_load(env_sensitive))
```

Comment:
Could use `distutils.util.strtobool` here.

Reply:
I thought about it but now that `distutils` is deprecated, I'd prefer to avoid it.

Reply:
Good point.